### DISCUSSION TITLE
kconfig.js file not exist, but run need it

### DISCUSSION THREAD

**Discussion by wangyufengx** [Kong/kong-manager#124] [Q&A]
![image](https://github.com/Kong/kong-manager/assets/30011407/798210fc-0929-4744-a066-692a88833590)

**Comment by sumimakito** [comment]
Hello @wangyufengx - Do you have Kong Gateway started in your local environment?

**Reply by wangyufengx** [reply]
no. i run it other server, but i want local test

**Reply by nekolab** [reply]
Maybe setting the environment variable `KONG_GUI_URL` to the address of the gateway server can solve your problem.
